src/hazmat/primecount.rs

```diff
@@ -0,0 +1,117 @@
+use crypto_bigint::{Uint, U256};
+
+/// Estimate the number of primes smaller than x using the asymptotic expansion of Li(x) with three terms. 𝜋(𝑥)/Li(x)
+/// converges to 1 from below, which means that the estimation of number of primes for the ranges relevant here is a
+/// slight overestimate.
+/// Uses the formula 𝜋(𝑥) ~ (x/ln(x)) * (1 + 1!/ln(x) + 2!/(ln(x)^2) + 3!/(ln(x)^3)).
+pub fn estimate_pi_x<const LIMBS: usize>(x: &Uint<LIMBS>) -> Uint<LIMBS> {
```

Comment:
May be useful to have two methods, for the upper bound (using `Li(x)`), and for the lower bound (using `x/ln(x)`)